microprofile-health/src/test/resources/org/jboss/eap/qe/microprofile/health/integration/file-props/dummy.in_maintenance

```diff
@@ -0,0 +1 @@
+2
```

Comment:
Would please consider the idea of adding a one line comment to make clear that numbers here are nothing else that placeholders or whatever?

Reply:
There can not be any coment. The file must contain value for the property which name is equal to the filename

Reply:
Sorry but I don't get this. Aren't you providing those values as Boolean?

Reply:
what about `a random string to be changed`?

Reply:
Sounds good to me, anything that would not lead to think about enumeration or some hidden logic.